app/models/post.rb

```diff
@@ -0,0 +1,6 @@
+class Post < ApplicationRecord
+  belongs_to :user
+  has_many :comments
+  validates :body, length: { maximum: 200 }
+  validates :user, presence: true
```

Comment:
#### Missing Validation
- [ ] `title` - Should not be able to receive an empty value. Missing Validation
- [ ] `body` - Should not be able to receive an empty value. Missing Validation